test/requests/support/rubocop_runner_test.rb

```diff
@@ -0,0 +1,26 @@
+# typed: true
+# frozen_string_literal: true
+
+require "test_helper"
+require "ruby_lsp/requests/support/rubocop_runner"
+
+module RubyLsp
+  module Requests
+    module Support
+      class RuboCopRunnerTest < Minitest::Test
+        def test_adds_raise_cop_error_flag_if_rubocop_version_supports_it
+          runner = RuboCopRunner.new("--lint")
+
+          assert(runner.instance_variable_get(:@options).fetch(:raise_cop_error))
+        end
+
+        def test_does_not_add_raise_cop_error_flag_if_rubocop_version_does_not_support_it
+          skip("fix this")
+          runner = RuboCopRunner.new("--lint")
```

Comment:
IMO, I think that's fine as long we test the full behaviour that depends on that, i.e. setup an environment that raises on that flag, and then test a real RuboCop failure and sure that we just process the files and not show any errors.

In order to test a real RuboCop failure, we can also programmatically stub other RuboCop methods so that our custom cop is included in the run as well, for example.

The alternative way to test it is to test against the real old version of RuboCop and that means forking or shelling out to a separate process, which is what we do on Tapioca. However, that is more time consuming and harder to get right, and seen as our implementation is tightly coupled with how RuboCop behaves anyway, I think stubbing RuboCop behaviour is fine.